.github/workflows/performance-benchmarks-naslib.yml

```
@@ -0,0 +1,198 @@
+          rm -fr cookies.txt confirm.txt
+        }
+        wget_gdrive 1sh8pEhdrgZ97-VFBVL94rI36gedExVgJ nb201_cifar10_full_training.pickle
+        cd ../../../
+    
+    - name: Cache nasbench201-cifar100 dataset
+      id: cache-nb201-cifar100-dataset
+      uses: actions/cache@v2
+      with:
+        path: NASLib/naslib/data/nb201_cifar100_full_training.pickle
+        
+        key: cache-nb201-cifar100
+
+    - name: Download nasbench201-cifar100 dataset
+      if: steps.cache-nb201-cifar100-dataset.outputs.cache-hit != 'true'
+      run: |
+
+        cd NASLib/naslib/data
+        function wget_gdrive {
+
+          GDRIVE_FILE_ID=$1
+          DEST_PATH=$2
+
+          wget --save-cookies cookies.txt 'https://docs.google.com/uc?export=download&id='$GDRIVE_FILE_ID -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1/p' > confirm.txt
+          wget --load-cookies cookies.txt -O $DEST_PATH 'https://docs.google.com/uc?export=download&id='$GDRIVE_FILE_ID'&confirm='$(<confirm.txt)
+          rm -fr cookies.txt confirm.txt
+        }
+        wget_gdrive 1hV6-mCUKInIK1iqZ0jfBkcKaFmftlBtp nb201_cifar100_full_training.pickle
+        cd ../../../
+
+    - name: Cache nasbench201-imagenet16 dataset
+      id: cache-nb201-imagenet16-dataset
+      uses: actions/cache@v2
+      with:
+        path: NASLib/naslib/data/nb201_ImageNet16_full_training.pickle
+        
+        key: cache-nb201-imagenet16
+
+    - name: Download nasbench201-imagenet16 dataset
+      if: steps.cache-nb201-imagenet16-dataset.outputs.cache-hit != 'true'
+      run: |
+
+        cd NASLib/naslib/data
+        function wget_gdrive {
+
+          GDRIVE_FILE_ID=$1
+          DEST_PATH=$2
+
+          wget --save-cookies cookies.txt 'https://docs.google.com/uc?export=download&id='$GDRIVE_FILE_ID -O- | sed -rn 's/.*confirm=([0-9A-Za-z_]+).*/\1/p' > confirm.txt
+          wget --load-cookies cookies.txt -O $DEST_PATH 'https://docs.google.com/uc?export=download&id='$GDRIVE_FILE_ID'&confirm='$(<confirm.txt)
+          rm -fr cookies.txt confirm.txt
+        }
+        wget_gdrive 1FVCn54aQwD6X6NazaIZ_yjhj47mOGdIH nb201_ImageNet16_full_training.pickle
+        cd ../../../
+
+
+    - name: Run performance benchmark
+      run: |
+        python benchmarks/run_naslib.py \
+          --path-to-kurobako "." \
+          --name-prefix "" \
+          --n-runs ${{ github.event.inputs.n-runs }} \
+          --n-jobs 6 \
+          --sampler-list '${{ github.event.inputs.sampler-list }}' \
+          --sampler-kwargs-list '${{ github.event.inputs.sampler-kwargs-list }}' \
+          --pruner-list '${{ github.event.inputs.pruner-list }}' \
+          --pruner-kwargs-list '${{ github.event.inputs.pruner-kwargs-list }}' \
+          --seed 0 \
+          --out-dir "out"
+
+    - uses: actions/upload-artifact@v2
+      with:
+        name: benchmark-report
+        path: |
+          out/result.json
```

The generated file name is not `result.json` but `results.json`.
```suggestion
          out/results.json
```